app/models/users.py

```diff
@@ -14,12 +14,14 @@ class TokenData(SQLModel):
     username: str | None = None
 
 
-class Users(SQLModel, table=True):
+class User(SQLModel, table=True):
```

Comment:
Should a user have an ID and when they were created?
Also it would be good to know when they last logged in so that we can monitor/delete in-active users